### PR TITLE
Add Rake tasks for deploy and merge down.

### PR DESCRIPTION
**Pull request by awong-dev**
This removes yet another chance for human error in deploy.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,10 +1,23 @@
-task :default => [ :"tests:ci" ]
+require "tmpdir"
```
Comment (llimllib) [inline Rakefile]: I don't think this line is necessary?

```
$ pry -v && pry
Pry version 0.10.3 on Ruby 2.1.2
[1] pry(main)> Dir.mktmpdir
=> "/var/folders/3y/xm71yyc94_7cmmjwncf5pqmc0000gn/T/d20160104-80537-nkv9y9"
```

File: Rakefile [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+# Clones the current git repository into a temp directory that shares the
+# object store. Also sets up a github remote.
+#
+# Commands can be run against this clone can be used to manipulate the
+# github remote without affecting your current checkout.
+def run_in_cloned_git
+  Dir.mktmpdir("vets-website-deploy") do |dir|
+    sh "git clone -l --single-branch -s . #{dir}"
+    sh "cd #{dir}"
+    sh "git remote | grep github && git remote remove github"
```
Comment (llimllib) [inline Rakefile]: This fails the whole `run_in_cloned_git` command if there isn't any remote called `github`, doesn't it?

File: Rakefile (hunk 1 above)
Comment (awong-dev) [inline Rakefile]: Dir.mktmpdir didn't exist for me w/o it...

File: Rakefile (hunk 2 above)
Comment (awong-dev) [inline Rakefile]: Nice catch! Ugh. I'll need to rethink that tomorrow when I have more brain juice.

Comment (awong-dev) [inline Rakefile]: if (git remote | grep -q '^github$'); then git remote remove github; fi

**Comment by awong-dev** [comment]
@llimllib PTAL

**Comment by llimllib** [comment]
LGTM, but that's just from reading it; no idea how to test it really

**Comment by awong-dev** [comment]
@llimllib Thanks for the review! I tested it locally by changing the target branches to master-test and production-test.  Verified it created the things...would be nice to have an automated test, but I don't know how to do that for Rakefiles :-/

Merging for now.

**Pull request merged**